dir-walker/Containerfile

```diff
@@ -0,0 +1,27 @@
+# Building the example app from source
+# This is a contrived example that traverses through all directories in a given path and returns the largest 10 files. 
+FROM golang:1.19 AS build-stage
+WORKDIR /app
+ADD dir-walker.go Makefile .
+RUN make build
+
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
+# This example uses rhel-coreos image from 4.13.0-rc.2
+FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d98795f7932441b30bb8bcfbbf05912875383fad1f2b3be08a22ec148d68607e
```

Comment:
We are keeping the url generic, something like https://github.com/openshift/rhcos-image-layering-examples/blob/master/libreswan/Containerfile#L3 . This is to avoid confusion where user may just copy paste and expect it to work.